src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj

```diff
@@ -19,8 +19,7 @@
     <PropertyGroup>
       <LibrariesTrimmedArtifactsPath>$(TargetDir)wasm-linked\</LibrariesTrimmedArtifactsPath>
       <!-- Link all assemblies -->
-      <ILLinkArgs>$(ILLinkArgs) -c link</ILLinkArgs>
-      <ILLinkArgs>$(ILLinkArgs) -u link</ILLinkArgs>
+      <ILLinkArgs>$(ILLinkArgs) --action link</ILLinkArgs>
```

Comment:
Can you help me understand how this change related to the bug?

Reply:
The linker changed in the new SDK and it no longer supports some of these switches.